web/src/input_pill.ts

```diff
@@ -403,9 +466,9 @@ export function create(opts) {
         },
 
         clear: funcs.removeAllPills.bind(funcs),
-        clear_text: funcs.clear_text,
-        is_pending: funcs.is_pending,
-        _get_pills_for_testing: funcs._get_pills_for_testing,
+        clear_text: funcs.clear_text.bind(funcs),
+        is_pending: funcs.is_pending.bind(funcs),
+        _get_pills_for_testing: funcs._get_pills_for_testing.bind(funcs),
```

Comment:
These changes to use `.bind` consistently would also be better done as a prep commit.